EventListener/ParamConverterListener.php

```diff
@@ -96,7 +96,7 @@ private function autoConfigure(\ReflectionFunctionAbstract $r, Request $request,
                 $configurations[$name]->setClass($param->getClass()->getName());
             }
 
-            $configurations[$name]->setIsOptional($param->isOptional());
+            $configurations[$name]->setIsOptional($param->isOptional() || null !== $param->getType() && $param->getType()->allowsNull());
```

Comment:
You should probably check that the `getType` method exist as it does not exist in old version of PHP (this is the reason tests are currently red :wink:)